homeassistant/helpers/config_validation.py

```diff
@@ -646,6 +646,17 @@ def string_with_no_html(value: Any) -> str:
     return str(value)
 
 
+def string_no_single_quoted_placeholders(value: Any) -> str:
```

Comment:
We only use this once, does it make sense to add it here or just at the place where we use it

Reply:
I'm happy to move it. I just followed the same pattern as `cv.string_with_no_html`, but that is used in `script/hassfest/icons.py` as well.

Reply:
Moved it. OK now?